stamp_generation/main_stamp_generation.py

```diff
@@ -0,0 +1,26 @@
+"""
+   This script starts program for stamp stamp_generation
+
+   Once the summarizer is ready, we can move
+   this script to main.py in stampify/
+
+   Command to run:
+   /stampify/stamp_generator$ python main_stamp_generation.py
+"""
+
+from stamp_generation import generator_input, stamp_generator
+
+contents_list = generator_input.get_sample_contents_list()
+
+# Output of summarizer
+
+stamp_pages = generator_input.get_sample_stamp_pages()
+
+generator = stamp_generator.StampGenerator(contents_list, stamp_pages)
+stamp = generator.generate_stamp()
+
+# final_generated_stamp.html has the generated amp-html stamp code
+
+f = open('final_generated_stamp.html', 'w')
+f.write(stamp)
+f.close()
```

Comment:
Prefer IO operation under try/catch

Reply:
Will look into it! Thanks for the suggestion.

Reply:
Since we are using the IO operation for write purpose it won't break if file is not found rather it will create one. Do we still need to use try/catch? Let me know if I understand it wrong.

Reply:
Updated!